test/kokoro/xds.sh

```diff
@@ -13,7 +13,14 @@ shopt -s extglob
 branch="${branch//[[:space:]]}"
 branch="${branch##remotes/origin/}"
 shopt -u extglob
-go build
+# Install a version of Go supported by gRPC for the new features, e.g.
+# errors.Is()
+sudo apt-get install -y wget
+wget https://go.dev/dl/go1.17.3.linux-amd64.tar.gz
```

Comment:
if `curl` is already in the VM, it would be preferable to use it.  Otherwise this `apt install` is another potential point of failure.

Also, should these things be retried for the same reason as `go build`?

Reply:
Done